docs/agent/changelog.md

```diff
@@ -6,6 +6,12 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.17.0 - \[2024-10-10\]
+
+- Added endpoint create/update/delete subcommands to the ngrok api command
+- Added support for policy phases like `on_tcp_connect`, `on_http_request` and `on_http_response`
+- Fix to support correct configuration for HTTP policies
```

Comment:
i feel like this line needs a little more detail to be useful. 
```suggestion
- Fixed a bug where we were incorrectly passing TLS as the endpoint proto rather than HTTP(S).
```